test/evergreen.yml

```diff
@@ -128,6 +128,15 @@ functions:
     type: setup
     params:
       directory: wiredtiger
+  "generate github token": &gen_github_token
+    command: github.generate_token
```

Comment:
The Github app has been provisioned in the "WiredTiger (develop)" Evergreen project to enable this command invocation.